backend/src/main/java/com/htecgroup/skynest/security/WebSecurity.java

```diff
@@ -11,6 +14,7 @@
 import org.springframework.security.config.annotation.web.configuration.WebSecurityConfigurerAdapter;
 import org.springframework.security.config.http.SessionCreationPolicy;
 import org.springframework.security.crypto.bcrypt.BCryptPasswordEncoder;
+import org.springframework.security.web.authentication.UsernamePasswordAuthenticationFilter;
 
 @EnableWebSecurity
 @Configuration
```

Comment:
This class is a good candidate to move the configuration from UserController

Reply:
I've moved it, but RolesAllowed always returns 403, and I made sure to test this with jsr250Enabled=true.